wherehows-backend/app/actors/ConfigUtil.java

```diff
@@ -74,7 +74,7 @@ static ProcessBuilder buildProcess(String etlJobName, long whEtlExecId, String c
         .add("-DCONTEXT=" + etlJobName)
         .add("-Dlogback.configurationFile=etl_logback.xml")
         .add("-DLOG_DIR=" + outDir)
-        .add("metadata.etl.Launcher")
+        .add("wherehows.common.jobs.Launcher")
```

Comment:
Maybe use Launcher.class.getCacnonicalName() instead of hard-coded strings for easier refactoring in the future?